prow/plugins/trigger/generic-comment.go

```diff
@@ -114,6 +115,17 @@ func handleGenericComment(c Client, trigger plugins.Trigger, gc github.GenericCo
 	if err != nil {
 		return err
 	}
+
+	for _, ps := range toTest {
+		groups := ps.GetGroups(gc.Body)
+		if groups != nil {
+			for i, group := range groups[1:] {
+				env := v1.EnvVar{Name: fmt.Sprintf("ARG%d", i), Value: group}
```

Comment:
I don't understand how ARG0=group results in the trigger time?

Reply:
eg regex for trigger field in prowjob:
```
trigger: "`(?mi)^/testcommand\s+([0-9])\s*$" 
```
so `groups[0]` will have the first matching group? that we put into `ARG0` env var and so on.

Reply:
made this PR as part of this discussion, so we might mean different things by `trigger time parameters` i just meant having to pass the comment parameters to the job when it triggers.

https://kubernetes.slack.com/archives/CDECRSC5U/p1560940499239400

Reply:
"Time" makes me think this is about timestamps but it seems like you want to be able to send arbitrary parameters to triggered jobs.

Can you update the PR description to remove the word time? As Steve mentioned in this thread let's see if we can get consensus on the design of this feature before we do anything with it.